frontend/testing/playwright/playwright.config.ts

```diff
@@ -118,6 +130,7 @@ export default defineConfig<ExtendedTestOptions>({
         TestNames.GIT_SYNC,
         TestNames.UI_EDITOR,
         TestNames.SETTINGS_MODAL,
+        TestNames.TEXT_EDITOR,
```

Comment:
Hva med å hente ut verdien av enum her direkte sånn som dette: `dependencies: [...Object.values(TestNames)]`. Da slipper vi å manuelt legge til apper testene som avhengigheter her.

Reply:
Smart! 
Men, met må nok gjøres slik for å ikke få sirkulære avhengigheter: 
```[...Object.values(TestNames).filter((testName) => testName !==TestNames.LOGOUT_AND_INVALID_LOGIN_ONLY)``` 😄